internal/coverage/profile.go

```diff
@@ -18,14 +18,15 @@ package coverage
 
 import (
 	"go/token"
+	"path/filepath"
 )
 
 // Profile is implemented as a map holding a slice of Block per each filename.
 type Profile map[string][]Block
 
 // IsCovered checks if the given token.Position is covered by the coverage Profile.
 func (p Profile) IsCovered(pos token.Position) bool {
-	blocks, ok := p[pos.Filename]
+	blocks, ok := p[filepath.FromSlash(pos.Filename)]
```

Comment:
I guess. This will not fix problem. Because `(c *Coverage) filePath()` should be fixed with `filepath.FromSlash`.

It can't open file with `%v/%v` path. And here (in IsCovered) we do not have coverage data.

Reply:
So. Can you please
1. Revert other changes (exept this one)
2. Add fix from comment above to filePath func
3. Add here debug point or loging for `p` and `pos.Filename` content. To see that is going on.

After that we can decide what to do next. Leave fix here, or add fix to `(c *Coverage) removeModuleFromPath`